Assets/PatchKit Patcher/Editor/PatcherManifestCreator.cs

```diff
@@ -37,9 +37,19 @@ private static void PostProcessBuild(BuildTarget buildTarget, string buildPath)
             string manifestPath = Path.Combine(Path.GetDirectoryName(buildPath), "patcher.manifest");
             string manifestContent = JsonConvert.SerializeObject(manifest, Formatting.Indented);
 
+            if (File.Exists(manifestPath))
+            {
+                File.Delete(manifestPath);
```

Comment:
Is it necessary? `File.WriteAllText` should overwrite file AFAIK

Reply:
Yeah, you're right, my mistake